.github/workflows/test.yml

```diff
@@ -30,6 +30,13 @@ jobs:
           # config: '/lint/config/changelog.yml'
     #     args: '.'
 
+
+    - name: checkout test suite
+      uses: actions/checkout@v4
+      with:
+        repository: eessi/test-suite
+        path: src
```

Comment:
Why don't we just do this without `path`, i.e. in the current workdir? It will still clone into a subidr `test-suite` by default right? No need to nest that further inside a `src` I'd think.